### PR TITLE
ASC-581-Rdkit library integration

### PR DESCRIPTION
**Pull request by Sneha-SR**
1. Rdkit library Integration.
2. Added an util file with function to render compound which can be invoked from client component which accepts smile as input
3. If it doesn't work from service file that can be copied component and used for rendering
4. example to invoke in component: <**div dangerouslySetInnerHTML={{ __html: getSmileStructure("CC([C@H](N)C(O)=O)C") }}></div>**

### CODE REVIEW

File: apps/web/components/Dashboard/LandingPage.tsx [hunk 1]
```diff
@@ -50,6 +61,7 @@ export default function LandingPage({ userData,
 
     return (
         <>
+            <Script src="https://unpkg.com/@rdkit/rdkit/dist/RDKit_minimal.js" strategy="beforeInteractive" />
```
Comment (himanshubexcelra) [inline apps/web/components/Dashboard/LandingPage.tsx:64]: This is wrong.

Comment (Sneha-SR) [reply apps/web/components/Dashboard/LandingPage.tsx:64]: with out this it was not working Himanshu

Comment (Sneha-SR) [reply apps/web/components/Dashboard/LandingPage.tsx:64]: but later I did some changes will comment and check this

Comment (himanshubexcelra) [reply apps/web/components/Dashboard/LandingPage.tsx:64]: You are adding it on lanfing page. 

It should be in root layout not on specific page.

Comment (Sneha-SR) [reply apps/web/components/Dashboard/LandingPage.tsx:64]: ok

**Pull request closed without merging**